CONTRIBUTING.md

```diff
@@ -169,10 +169,14 @@ Environment variables:
 - You **need** to set ESX and either VM_IP (in which case we'll use 1 VM) or
 both VM1 and VM2 environment variables
 
+- You need to set `DOCKER_HUB_REPO` as environment variable to complete our move to use [managed plugin](https://github.com/vmware/docker-volume-vsphere/blob/master/plugin/Makefile#L30).
+
+**Note**: You need to manually remove older rpm/deb installation from your test VMs. With [PR 1163](https://github.com/vmware/docker-volume-vsphere/pull/1163), our build/deployment script start using managed plugin approach.
```

Comment:
I am going to generate a new PR to talk about upgrade scenario, this is just a note for code contributor for our repo.

I will add the sequence you have mentioned at the FAQ page under upgrade section.